packages/contracts-bedrock/src/L1/OPContractsManager14.sol

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { OPContractsManager } from "./OPContractsManager.sol";
+// Libraries
+import { Blueprint } from "src/libraries/Blueprint.sol";
+import { Claim, GameType, GameTypes } from "src/dispute/lib/Types.sol";
+// Interfaces
+import { IBigStepper } from "interfaces/dispute/IBigStepper.sol";
+import { IDisputeGame } from "interfaces/dispute/IDisputeGame.sol";
+import { ISystemConfig } from "interfaces/L1/ISystemConfig.sol";
+import { IDisputeGameFactory } from "interfaces/dispute/IDisputeGameFactory.sol";
+import { IFaultDisputeGame } from "interfaces/dispute/IFaultDisputeGame.sol";
+import { IPermissionedDisputeGame } from "interfaces/dispute/IPermissionedDisputeGame.sol";
+import { IProtocolVersions } from "interfaces/L1/IProtocolVersions.sol";
+import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
+import { IProxyAdmin } from "interfaces/universal/IProxyAdmin.sol";
+
+///  @title OPContractsManager14
+///  @notice Represents the new OPContractsManager for Upgrade 14
+contract OPContractsManager14 is OPContractsManager {
+    function version() public pure override returns (string memory) {
+        return string.concat(super.version(), "+upgrade14.1");
+    }
+
+    // TODO: Review required arguments
```

Comment:
TODO: Not sure that we need all of these arguments since the functionality is stripped down

Reply:
I think there's only 1 arg we can cut (protocolVersions) - done